lib/import/mobility.js

```diff
@@ -0,0 +1,192 @@
+  /**
+   * Save a splice of array of mobility records.
+   * @param{array} ary - array of mobility objects
+   */
+    function save(ary) {
+      if (ary.length > 0) {
+        if (verbose) { console.log(debug_msg, ary.length);}
+        Mobility.insertMany(ary.splice(0, 1000))
+        .then(function() {
+          save(ary);
+        })
+        .catch(function(err) {
+          if (err) {return reject(err);}
+        });
+      } else {
+        resolve();
+      }
+    }
+    if (verbose) { console.log(debug_msg, ary.length);}
+    save(ary);
+  });
+}
+
+/**
+ * Return list of mobility files in directory.
+ *
+ * @param{string} dir - directory where mobility files are stored
+ * @return{promise} Fulfilled once all files are in list
+ */
+function get_mobility_files(dir) {
+  return new Promise(function(resolve, reject) {
+    fs.readdir(dir, function(err, list) {
+      if (err) { console.log(err); return reject(err);}
+      list = list.filter(function(e) {
+        return e.match(/.csv/);
+      });
+      resolve(list);
+    });
+  });
+}
+
+/**
+ * Import mobility from CSV to Mongo.
+ *
+ * @param{string} dir - directory where mobility csv files exist.
+ * @param{string} country_code - lowercase ISO 3166-1 alpha-2 country code
+ *   https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+ * @param{bool} verbose - whether to log messages while importing.
+ * @return{promise} Fulfilled once all records are saved.
+ *
+ * Needs to be connected to mongo before calling.
+ */
+function import_mobility(dir, country_code, verbose) {
+  var all_done = [];
+  return get_mobility_files(dir)
+  .then(function(list) {
+    list.forEach(function(csv_file) {
+      var promise = new Promise(function(resolve) {
+        import_file(dir + '/' + csv_file, country_code, verbose)
+        .then(function() {
+          resolve();
+        });
+      });
+      all_done.push(promise);
+    });
+    return Promise.all(all_done);
+  });
+}
+
+/** Main function for when this module is called directly as a script. */
+function main() {
+  var parser = new ArgumentParser({
+    version: '0.0.1',
+    addHelp: true,
+    description: 'Import mobility docs into db from a directory of csv files.'
+  });
+  parser.addArgument(
+    ['-c', '--country_code'],
+    {
+      help: 'ISO 3166 alpha-2 two letter code for the country.'
+    }
+  );
+  parser.addArgument(
+    ['-d', '--dir'],
+    {
+      help: 'Name of directory'
+    }
+  );
+
+  parser.addArgument(
+    ['--verbose'],
+    {
+      help: 'Print debug'
+    }
+  );
+
+  var args = parser.parseArgs();
+
+  var mobility_dir = args.dir ||
+      'data/mobility/' + args.country_code;
+
+  var mongodb = config.database;
+  mongoose.connect(mongodb, function(err) {
+    if (err) {
+      throw err;
+    }
+
+    import_mobility(mobility_dir, args.country_code, args.verbose)
+    .then(function() {
+      return process.exit();
+    }).catch(function(err) { console.log(err); process.exit();});
```

Comment:
When programs encounter errors, they should exit with a non-zero exit code. That's how external scripts and such can get signaled that something went wrong. This would be useful if we built up more majicbox infrastructure to automatically load new mobility records, for example.

So, just make this `process.exit(1)`.